build.ps1

```diff
@@ -207,12 +207,11 @@ Foreach ($target in $targets) {
     $specOutDir = [io.path]::combine($OUT_DIR, $target.Name)
     CleanDir $specOutDir
 
-    BuildDummyApp $specOutDir $target
     BuildServer $SERVER_SRC_DIR $specOutDir $target
-    BuildTool rvn $RVN_SRC_DIR $specOutDir $target
-    BuildTool drtools $DRTOOL_SRC_DIR $specOutDir $target
-    BuildTool migrator $MIGRATOR_SRC_DIR $specOutDir $target
-    BuildDebug $DEBUG_SRC_DIR $specOutDir $target
+    BuildTool rvn $RVN_SRC_DIR $specOutDir $target $true
+    BuildTool drtools $DRTOOL_SRC_DIR $specOutDir $target $false
+    BuildTool migrator $MIGRATOR_SRC_DIR $specOutDir $target $true
+    BuildTool debug $DEBUG_SRC_DIR $specOutDir $target $true
```

Comment:
the flag does not seem to be in use in `BuildTool()`?

Reply:
I copied it back 1:1 from 6.0 branch, didn't want to make any new additions